src/test/VisitorTest.java

```diff
@@ -0,0 +1,46 @@
+package test;
+
+import main.*;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class VisitorTest {
+    TaxVisitor taxCalculator = new TaxVisitor();
+    TaxHolidayVisitor taxHolidayCalculator = new TaxHolidayVisitor();
+
+
+    Necessity milk = new Necessity(1.99);
```

Comment:
I would probably inline all of these, simply because it keeps the data (e.g. 1.99) within the test, which allows it to be quickly compared to the expectation. So although this approach is drier, it requires scanning the test in two places in order to properly understand the test.